### PR TITLE
capetown

### PR DESCRIPTION
**Pull request by visseho**
folder created

### CODE REVIEW

File: _data/2018/capetown/faculty.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Local faculty
```
Comment (lukebaker) [inline _data/2018/capetown/faculty.yml]: This is not valid YAML. See the [primary site file as an example](https://github.com/compsocialscience/summer-institute/blob/1d69e8edab5afa2a21ec45b8d94df69d5bcc5a07/_data/2018/faculty.yml).

File: _data/2018/capetown/faculty.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- name: Viss??ho Adjiwanou
+  image: 
+  bio: "Viss??ho Adjiwanou is a Senior Lecturer in Demography and Quantitative Methods at the University of Cape Town (South Africa), and adjunct professor at the Department of Demography at the Universit?? de Montr??al (Canada). His research interests include maternal and reproductive health, family dynamics, and female employment in sub-Saharan Africa. Viss??ho is the chair of the Panel on Computational Social Science at the Union for African Population Studies."
+  website: https://www.researchgate.net/profile/Visseho_Adjiwanou 
+
+
+International faculty
```
Comment (lukebaker) [inline _data/2018/capetown/faculty.yml]: This line is also not allowed here in YAML.

File: _data/2018/capetown/speakers.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+International speakers (By video)
```
Comment (lukebaker) [inline _data/2018/capetown/speakers.yml]: This line is not valid here in YAML.

File: _data/2018/capetown/speakers.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  image: /assets/images/cynthia.jpg
+  bio: Cynthia Rudin is an Associate Professor of computer science, electrical and computer engineering, and statistics at Duke University, and directs the Prediction Analysis Lab.
+  website: https://users.cs.duke.edu/~cynthia/
+  
+- name: Duncan Watts
+  image: /assets/images/duncan.jpg
+  bio: Duncan Watts is a Principal Researcher at Microsoft Research and a founding member of the MSR-NYC lab. He is also an AD White Professor at Large at Cornell University.
+  website: https://www.microsoft.com/en-us/research/people/duncan/
+  
+Local speakers
```
Comment (lukebaker) [inline _data/2018/capetown/speakers.yml]: This line is not valid YAML.

**Comment by visseho** [comment]
Just very quickly. Is that because I used accent in my name the problem ? Vissého
Or, because I don’t have the image from the other speakers.
I have made the change on my folder.

**Comment by lukebaker** [comment]
> Just very quickly. Is that because I used accent in my name the problem ? Vissého
> Or, because I don’t have the image from the other speakers.
> I have made the change on my folder.

It was due to the additional lines for "Local faculty", etc. Not because of the accent in your name. While the accent wasn't showing up properly, it wasn't causing any problems besides being rendered as ??. I responded to you in Slack as well. Because it sounded like you were eager to get this in place for your talk, I went ahead and make the fixes in https://github.com/compsocialscience/summer-institute/pull/38/commits/8df9341620dd7555020eb9bd49bd2107fe125fae and https://github.com/compsocialscience/summer-institute/pull/38/commits/2b80800529fead9e659d01968785e720d07ccdbf and merged this pull request into master. You should see the capetown site now: https://compsocialscience.github.io/summer-institute/2018/capetown/

**Pull request merged**